### ISSUE TITLE
Probably SSL problem

### ISSUE THREAD

**Issue by moritzsternemann** [defaultnamehere/zzzzz#13]
I started the fetcher using

```
python2.7 fetcher.py
```

After two failed attempts I quit the script usinc CTRL+C.
It showed this Traceback:

```
>
Got error from request, restarting...
>
Got error from request, restarting...
>
^CTraceback (most recent call last):
  File "fetcher.py", line 176, in <module>
    f.start_request()
  File "fetcher.py", line 93, in start_request
    resp = self.make_request()
  File "fetcher.py", line 49, in make_request
    response_obj = requests.get(url, params=self.params, headers=self.REQUEST_HEADERS)
  File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 67, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 53, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 468, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 576, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/adapters.py", line 376, in send
    timeout=timeout
  File "/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py", line 559, in urlopen
    body=body, headers=headers)
  File "/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py", line 345, in _make_request
    self._validate_conn(conn)
  File "/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py", line 784, in _validate_conn
    conn.connect()
  File "/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/connection.py", line 252, in connect
    ssl_version=resolved_ssl_version)
  File "/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/contrib/pyopenssl.py", line 300, in ssl_wrap_socket
    cnx.do_handshake()
  File "/usr/local/lib/python2.7/site-packages/OpenSSL/SSL.py", line 1441, in do_handshake
    result = _lib.SSL_do_handshake(self._ssl)
KeyboardInterrupt
```

**Comment by emilstahl** [comment]
Try

`pip install requests[security]`

**Comment by moritzsternemann** [comment]
Problem solved using help of #29
